lib/Elastica/ResultSet.php

```diff
@@ -15,117 +15,45 @@
 class ResultSet implements \Iterator, \Countable, \ArrayAccess
 {
     /**
-     * Class for the static create method to use.
-     *
-     * @var string
-     */
-    protected static $_class = 'Elastica\\ResultSet';
-
-    /**
      * Results.
      *
-     * @var array Results
+     * @var Result[] Results
      */
-    protected $_results = array();
+    private $_results = array();
```

Comment:
Yeah, that is what I mean by my note, that it should be added to the CHANGELOG. I'm ok with the BC break.